setup.py

```diff
@@ -18,13 +18,15 @@
 with open('LICENSE') as f:
     license = f.read()
 
+with open('requirements.txt') as f:
+    requirements = f.read()
```

Comment:
I suspect this has to be `f.read().splitlines()`, but I'm not sure.

Reply:
you are correct, fixed in commit https://github.com/WyLyu/UPOsHam/pull/58/commits/16cab89b5f859e289d155defdb8d86bfa62f0e81